src/views/partials/head.html

```diff
@@ -4,4 +4,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Webpack 5 Project Starter</title>
+    <link href="@images/favicon.ico" rel="icon" />
+    <link href="@styles/style.scss" rel="stylesheet" />
+    <script src="@scripts/global.js" defer="defer"></script>
```

Comment:
you can use the source files of styles, scripts, images etc. directly in the HTML using the relative path to HTML file or an alias specified in Webpack config